.evergreen/config-atlas.yml

```diff
@@ -606,10 +606,9 @@ axes:
       - id: ubuntu1604
         display_name: "Ubuntu 16.04"
         run_on: ubuntu1604-small
-      # https://jira.mongodb.org/browse/RUBY-2758
-      # - id: ubuntu1404
-      #   display_name: "Ubuntu 14.04"
-      #   run_on: ubuntu1404-small
+      - id: ubuntu1404
```

Comment:
I am sure I will be confused when id ubuntu1404 while it is actually ubuntu1604. Can we change the id? Or just use existing 1604?

Reply:
So just switch everything to ubuntu1604?

Reply:
As I understand the changes, now both ubuntu1404 and and ubuntu1604 are based on `ubuntu1604-small`. So, they are basically the same, aren't they? If they are the same, I would rather stick with just one. If they are different, I would rename `ubuntu1404` to something that better reflects what it is.